test/integration/transactionsAPITests.js

```diff
@@ -37,133 +37,170 @@ const MAX_BODY_MB = 1
 const MAX_BODY_SIZE = MAX_BODY_MB * 1024 * 1024
 
 describe('API Integration Tests', () => {
-  const { SERVER_PORTS } = constants
-  const LARGE_BODY = Buffer.alloc(MAX_BODY_SIZE, '1234567890').toString()
-
-  const requestDoc = {
-    path: '/api/test',
-    headers: {
-      'header-title': 'header1-value',
-      'another-header': 'another-header-value'
-    },
-    querystring: 'param1=value1&param2=value2',
-    body: '<HTTP body request>',
-    method: 'POST',
-    timestamp: '2014-06-09T11:17:25.929Z'
-  }
-
-  Object.freeze(requestDoc)
-
-  const responseDoc = {
-    status: '200',
-    headers: {
-      header: 'value',
-      header2: 'value2'
-    },
-    body: '<HTTP response>',
-    timestamp: '2014-06-09T11:17:25.929Z'
-  }
-
-  Object.freeze(responseDoc)
-
-  const transactionData = {
-    _id: '111111111111111111111111',
-    status: 'Processing',
-    clientID: '999999999999999999999999',
-    channelID: '888888888888888888888888',
-    request: requestDoc,
-    response: responseDoc,
-
-    routes: [{
-      name: 'dummy-route',
-      request: requestDoc,
-      response: responseDoc
-    }
-    ],
-
-    orchestrations: [{
-      name: 'dummy-orchestration',
-      request: requestDoc,
-      response: responseDoc
-    }
-    ],
-    properties: {
-      prop1: 'prop1-value1',
-      prop2: 'prop-value1'
-    }
-  }
-
-  Object.freeze(transactionData)
-
+  let SERVER_PORTS, LARGE_BODY, requestDocMain, responseDocMain, requestDoc, responseDoc, transactionData
   let authDetails = {}
   let channel
   let channel2
   let channel3
+  let channelDoc
+  let channel2Doc
+  let channel3Doc
+
+  before(async () => {
+    SERVER_PORTS = constants.SERVER_PORTS
+    LARGE_BODY = Buffer.alloc(MAX_BODY_SIZE, '1234567890').toString()
+
+    // start the server before using the mongo connection
+    await promisify(server.start)({ apiPort: SERVER_PORTS.apiPort })
+
+    await testUtils.deleteChuckedPayloads()
```

Comment:
@BMartinos is `Chucked` a term? I've seen it a couple times - thought it was a typo at first - now I'm suspecting it means something else :|

Reply:
@MattyJ007 It is a term, but unfortunately not the correct one in these cases. This is a typo that was missed and that has been carried over to other parts of the code. Will correct the spelling :+1:

Reply:
Done in d465a8d